src/provider/Provider.ts

```diff
@@ -11,7 +11,6 @@ import {
 } from '../@types/'
 import { noZeroX } from '../utils/ConversionTypeHelper'
 import fetch from 'cross-fetch'
-import { DownloadResponse } from '../@types/DownloadResponse'
```

Comment:
more unused imports above imported `from '../utils'`

Reply:
GitHub still sees them, just couldn't comment on those lines as they were not changed:

<img width="833" alt="Screen Shot 2022-04-14 at 17 16 38" src="https://user-images.githubusercontent.com/90316/163431022-95a67e5c-6699-4674-8474-63cf38974fbe.png">